main.go

```diff
@@ -76,25 +80,18 @@ func main() {
 		}
 	}()
 
-	pub := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
As URLs contain the current directory and autoindex is designed as a single page application, this is required to serve `index.html` by default.